pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -1896,3 +1932,48 @@ func getClusterRegion(cd *hivev1.ClusterDeployment) string {
 	}
 	return regionUnknown
 }
+
+func LoadReleaseImageVerifier(config *rest.Config) (verify.Interface, error) {
+	ns := os.Getenv(constants.HiveReleaseImageVerificationConfigMapNamespaceEnvVar)
+	name := os.Getenv(constants.HiveReleaseImageVerificationConfigMapNameEnvVar)
```

Comment:
There are nighty ClusterImageSets in all dedicated hive shards. Am I correct that these will not pass verification?

If so we may need a more complicated change, as implemented this looks to be a global setting. We might need to let users annotate their ClusterImageSets to disable, and coordinate with SD on how to get that done, and get hive config updated to enable.

Reply:
> There are nighty ClusterImageSets in all dedicated hive shards. Am I correct that these will not pass verification?

i can check, but afaik all nightlies published by ART are signed and published like any other release release-images.

> as implemented this looks to be a global setting. We might need to let users annotate their ClusterImageSets to disable

for verification to be useful, it kind of has to be global and any step to skip it by modifying the object that we are modifying seems counter productive.
if we needed a way to define exclusions, it should probably be in HiveConfig. (hopefully explicit list, because selector based api has same problem of annotation on the object being verified)

Reply:
Ok please check, if the nightlies are ok then we're good.

If not, ClusterImageSet creation is intended to be a way for management cluster admins to control what releases can be installed, so being able to disable at that level should be about the same privilege level as editing HiveConfig, but a lot easier for the appsre job the publishes those to implement.

Reply:
> If not, ClusterImageSet creation is intended to be a way for management cluster admins to control what releases can be installed, so being able to disable at that level should be about the same privilege level as editing HiveConfig, but a lot easier for the appsre job the publishes those to implement.

for me the people that are allowed to manage image set are separate from Hive (the service) . admins of cluster image sets are still users and not admin of Hive.
the admins of Hive service should be the one caring about not running malicious code.

the goal of release image verification is to verify the cluster image sets are good, and having things on that object that disable verification does not sound good.

Reply:
> ClusterImageSet was meant for the admins of the hive service to control what releases can be installed, but lets not worry about it until we know nightlies are not signed with the right key.

that's not great if that was the intention. 🤷🏽 

checked that nightly release images are signed with same trust as the released release-images